starter-code/apple pie/styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+    
+}
+
+header {
+    display: relative;
+    top: 0;
+    left: 0;
+    width: cover;
+    
+    margin-block-end: 20px;
+    border-bottom: 2px solid grey;
+}
+
+.header {
+    background-image: url(images/apple-pie.jpg)
+}
+
+h1 {
+    color: white;
+    text-align: center;
+    position: relative;
+}
+
+.header_paragraphe {
```

Comment:
The reason these css properties were not applied is because in your HTML you have `id="header_paragraphe"` but here you're calling it as a class. 

so you need to change this to `#header_paragraphe`